src/inputs/Checkbox.stories.tsx

```diff
@@ -16,14 +16,16 @@ export function Checkboxes() {
         <h2 css={Css.mb1.$}>Basic Checkboxes</h2>
         <div css={Css.dg.gap1.$}>
           <Checkbox
+            selected={false}
             onChange={action("onChange")}
             onFocus={action("onFocus")}
             onBlur={action("onBlur")}
             label="Default"
           />
-          <Checkbox onChange={action("onChange")} selected label="Selected" />
-          <Checkbox onChange={action("onChange")} indeterminate label="Indeterminate" />
-          <Checkbox onChange={action("onChange")} disabled label="Disabled" />
+          <Checkbox onChange={action("onChange")} selected={false} label="Selected" />
+          <Checkbox onChange={action("onChange")} selected="indeterminate" label="Indeterminate" />
+          <Checkbox onChange={action("onChange")} selected={false} disabled label="Disabled while unselected" />
+          <Checkbox onChange={action("onChange")} selected={true} disabled label="Disabled while selected" />
```

Comment:
Where do we expect `selected` will be coming from? A database field? Manually set as a prop by a developer? Form State?

Just wondering how we programmatically get from some value to `true | false | "indeterminate"`

Reply:
Yeah, it will nearly always come from a database boolean field of like "is this enabled yes/no".

I.e. I don't think I've ever seen indeterminate used for anything other than "partially selected rows in a table of parent/child selectable rows" :-) so I'm not too worried about how it would show up in other "just a regular CRUD form" places.